tst/EnergyPlus/unit/HVACSizingSimulationManager.unit.cc

```diff
@@ -136,6 +137,8 @@ class HVACSizingSimulationManagerTest : public EnergyPlusFixture
 TEST_F(HVACSizingSimulationManagerTest, WeatherFileDaysTest3)
 {
 
+    FluidProperties::GetFluidPropertiesData(*state);
```

Comment:
Again, here it would be good to add anything common to all the unit tests into the custom fixture.

Reply:
During unit test updating I couldn't call GetFluidPropertiesData more than once. I fixed that so now that function can be called multiple times without issue. So yes, this one can be moved to the fixture. And regarding my last comment, the init_state can be moved to the fixture as long as the getInputs in init_state can be called multiple times (to update unit tests with idf snippets). I feel like I need a wiki page.

Reply:
A discussion would be good since init_state will affect code and unit testing going forward. I did add init_state to process_idf for unit tests with idf snippets. The question is using init_state in the fixture vs process_idf vs unit test and what that means going forward.